cryptocurrencies-service/src/services/cryptocurrency-service.ts

```diff
@@ -66,7 +66,9 @@ export const getCryptocurrencyById = async (id: string): Promise<any> => {
     return cryptocurrency;
 }
 
-export const getCryptocurrencies = async (limit = 10, offset = 0): Promise<Cryptocurrency[]> => {
+export const getCryptocurrencies = async (limit = 10, offset = 0, searchText = [""]): Promise<Cryptocurrency[]> => {
+    const whereClause = searchText.map(text => `regex(?symbol, "${text}", "i")`).join(" || ");
```

Comment:
Rename this variable and use the same statements we used within `getCryptocurrenciesInfo` (`CONTAINS`, `LCASE`).